fastapi/openapi/utils.py

```diff
@@ -187,7 +188,7 @@ def get_openapi_path(
                         response.setdefault("content", {}).setdefault(
                             "application/json", {}
                         )["schema"] = response_schema
-                    response.setdefault("description", "Additional Response")
+                    response.setdefault("description", http.client.responses[int(additional_status_code)])
```

Comment:
Probably worth using `http.client.responses.get(int(additional_status_code), "Additional Response")` in case someone uses a custom response code

Reply:
Thanks your review. Done as what you suggest.

Reply:
Thanks @dmontagu ! Good idea.